sunpy/map/sources/soho.py

```diff
@@ -96,7 +95,66 @@ def _supported_observer_coordinates(self):
     @classmethod
     def is_datasource_for(cls, data, header, **kwargs):
         """Determines if header corresponds to an EIT image"""
-        return header.get('instrume') == 'EIT' or header.get('telescop') == 'Extreme-ultraviolet Imaging Telescope (EIT)'
+        return (header.get('instrume') == 'EIT' or header.get('telescop') == 'Extreme-ultraviolet Imaging Telescope (EIT)'  and header.get('level') is None)
+
+
+class EITL1Map(EITMap):
+    """
+    SOHO EIT L1 Image Map.
+
```

Comment:
Add a sentence on why we need a specific L1 map source for future archeologists?

Reply:
Added.